src/accessControl/acp/index.test.js

```diff
@@ -380,118 +401,112 @@ describe("AcpAccessControlStrategy", () => {
+        (p) => acpFns3.setAllowModes(p, createAcpMap(true, true)),
+        (p) => acpFns3.setAllOfRuleUrl(p, editorsPolicyRule)
       );
       const policyDataset = chain(
         mockSolidDatasetFrom(policyResourceUrl),
         (d) => setThing(d, editorsPolicyRule),
         (d) => setThing(d, policy)
       );
-      const editorsAC = chain(
-        acpFns.createControl(),
-        (ac) => acpFns.addPolicyUrl(ac, editorsPolicyUrl),
-        (ac) => acpFns.addPolicyUrl(ac, editorsPolicyUrl)
-      );
+      const editorsAC = acpFns3.mockAcrFor(datasetWithAcrUrl);
       jest.spyOn(scFns, "getSolidDataset").mockImplementation(() => {
         throw new Error("404");
       });
-      jest.spyOn(acpFns2, "getPolicy").mockReturnValue(policy);
+      jest.spyOn(acpFns3, "getPolicy").mockReturnValue(policy);
       const policyResource = chain(policyDataset, (d) =>
-        acpFns.addMockAcrTo(d)
+        acpFns3.addMockAcrTo(d)
       );
-      const datasetWithAcrEditors = chain(datasetWithAcr, (d) =>
-        acpFns.setControl(d, editorsAC)
+      const datasetWithAcrEditors = acpFns3.addMockAcrTo(
+        datasetWithAcr,
+        editorsAC
       );
       const datasetWithAcrControlAccess = chain(
         datasetWithAcrEditors,
-        (d) => acpFns.addAcrPolicyUrl(d, controlAccessPolicyUrl),
-        (d) => acpFns.addMemberAcrPolicyUrl(d, controlAccessPolicyUrl)
+        (d) => acpFns3.addAcrPolicyUrl(d, controlAccessPolicyUrl),
+        (d) => acpFns3.addMemberAcrPolicyUrl(d, controlAccessPolicyUrl)
       );
       jest
-        .spyOn(acpFns2, "saveAcrFor")
+        .spyOn(acpFns3, "saveAcrFor")
         .mockResolvedValueOnce(datasetWithAcrEditors)
         .mockResolvedValue(datasetWithAcrControlAccess);
       jest
-        .spyOn(acpFns2, "getSolidDatasetWithAcr")
+        .spyOn(acpFns3, "getSolidDatasetWithAcr")
         .mockImplementation(() => policyResource);
       jest
         .spyOn(scFns, "saveSolidDatasetAt")
         .mockImplementation(() => policyResource);
       jest.spyOn(scFns, "deleteFile").mockResolvedValue("deleted");
 
-      await expect(await acp.removeAgentFromPolicy(webId, "editors")).toEqual({
+      expect(await acp.removeAgentFromPolicy(webId, "editors")).toEqual({
         response: datasetWithAcrControlAccess,
       });
-      expect(acpFns2.saveAcrFor).toHaveBeenCalledWith(datasetWithAcrEditors, {
+      expect(acpFns3.saveAcrFor).toHaveBeenCalledWith(datasetWithAcrEditors, {
         fetch,
       });
-      expect(acpFns2.saveAcrFor).toHaveBeenCalledWith(
+      expect(acpFns3.saveAcrFor).toHaveBeenCalledWith(
         datasetWithAcrControlAccess,
         {
           fetch,
         }
       );
-      expect(acpFns2.saveAcrFor).toHaveBeenCalledWith(
+      expect(acpFns3.saveAcrFor).toHaveBeenCalledWith(
         datasetWithAcrControlAccess,
         {
           fetch,
         }
       );
     });
     it("deletes rule and policy is agent is the last one on the list", async () => {
```

Comment:
Heh, sorry for going outside of the scope of this PR, but typo was such an easy fix that I couldn't resist :)

```suggestion
    it("deletes rule and policy if agent is the last one on the list", async () => {
```